services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -594,6 +594,23 @@ async def list_all_projects_by_uuid_for_user(self, user_id: int) -> List[str]:
                 result.append(row[0])
             return list(result)
 
+    async def update_project_without_enforcing_checks(
```

Comment:
ok, prefer like this. although I would try not to re-create too many functions that do the same, that create a lot of code duplication. ok.